lib/mrsk/cli/app.rb

```diff
@@ -124,6 +124,35 @@ def containers
     on(MRSK.hosts) { |host| puts_by_host host, capture_with_info(*MRSK.app.list_containers) }
   end
 
+  desc "stale_containers", "Detect app stale containers"
+  option :stop, aliases: "-s", type: :boolean, default: false, desc: "Stop the stale containers found"
+  def stale_containers
+    with_lock do
+      stop = options[:stop]
+
+      on(MRSK.hosts) do |host|
+        roles = MRSK.roles_on(host)
+
+        roles.each do |role|
+          stale_versions = \
```

Comment:
Handled in https://github.com/mrsked/mrsk/pull/183/commits/12e3a562c4cae631045a4952ce6c197d40af93f7
The reason to put it in the SSHKIT layer is that otherwise, we can't leverage the `on(hosts)` parallelism.